admin/controllers/attendance.js

```diff
@@ -116,55 +118,66 @@ const downloadResults = async (req, res, next) => {
   csvStream.end()
 }
 
-const getSubmitAttendance = async (req, res, next) => {
-  res.locals.pageTitle = 'Attendance register'
+const getReviewPupilDetails = async (req, res, next) => {
+  res.locals.pageTitle = 'Review pupil details'
+  req.breadcrumbs('Headteacher\'s declaration form', '/attendance/declaration-form')
   req.breadcrumbs(res.locals.pageTitle)
-  const pupils = await pupilDataService.sqlFindPupilsByDfeNumber(req.user.School)
+  const pupils = await headteacherDeclarationService.findPupilsForSchool(req.user.School)
   if (!pupils) {
     throw new Error('No pupils found')
   }
+  pupils.sort((a, b) => {
```

Comment:
@danwilliams can you add the sorting and the identification flag call in a presenter (under helpers) so that the controller is intact in term of actual processing?
This will also make the sorting testable.